pom.xml

```diff
@@ -12,14 +12,14 @@
 
 	<groupId>life.qbic</groupId>
 	<artifactId>omero-client-lib</artifactId>
-	<version>1.1.0-SNAPSHOT</version>
+	<version>1.1.0</version>
```

Comment:
```suggestion
	<version>1.2.0-SNAPSHOT</version>
```

Reply:
Maybe merge it locally into development. That does not change the release/1.1.0 branch